functional-tests/fn_bridge_deposit_reclaim_drt_seen.py

```diff
@@ -0,0 +1,292 @@
+import time
+
+import flexitest
+from bitcoinlib.services.bitcoind import BitcoindClient
+from strata_utils import (
+    deposit_request_transaction,
+    get_address,
+    get_balance,
+    get_balance_recovery,
+    get_recovery_address,
+    take_back_transaction,
+)
+
+from constants import (
+    DEFAULT_ROLLUP_PARAMS,
+    DEFAULT_TAKEBACK_TIMEOUT,
+    SATS_TO_WEI,
+    UNSPENDABLE_ADDRESS,
+)
+from entry import BasicEnvConfig
+from utils import get_bridge_pubkey, get_logger
+
+# Local constants
+ROLLUP_DEPOSIT_AMOUNT = DEFAULT_ROLLUP_PARAMS["deposit_amount"]
+STRATA_DEPOSIT_LOWER_BOUND = ROLLUP_DEPOSIT_AMOUNT * SATS_TO_WEI
+
+
+@flexitest.register
+class BridgeDepositReclaimDRTSeenTest(flexitest.Test):
+    """
+    A test class for reclaim path scenarios of bridge deposits.
+
+    It tests the functionality of broadcasting a deposit request transaction (DRT) for the bridge
+    and verifying that the reclaim path works as expected.
+
+    In the test we stop one of the bridge operators to prevent the DRT from being processed.
+    However, we let the operator be aware of the DRT before stopping it.
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        ctx.set_env(BasicEnvConfig(pre_generate_blocks=101))
```

Comment:
For the purposes of this test, you might want to set `auto_generate_blocks` to `False` so that there is more granular control over how many blocks are generated and when.

Reply:
Done in 2b2cc8e6db5166e7342d03ae12c41dfa8917c1ac